docs/source/userguide/command_line.rst

```diff
@@ -1,508 +1,520 @@
+    geoips list interfaces -i
 
-The command above would list all GeoIPS Plugin Packages with information including their
-package name, docstring, and current version number. For a listing of what columns you
-can filter by, run ``geoips list <cmd_name> --columns help``.
+Both the general and implemented outputs can
+be filtered by package with ``--package_name`` or ``-p``.
 
-.. _geoips_list_interface:
+For example:
 
-:ref:`geoips list interface <geoips_list_interface>`
+.. code-block:: bash
 
-``list <interface_name>`` is a list sub-command which retrieves a listing of implemented
+    geoips list interfaces
 
-plugins of a certain interface. This can also be applied to a certain GeoIPS package.
-Information included when calling this command is:
+or
 
-    * GeoIPS Package
-    * Interface Name
-    * Interface Type
-    * Family
-    * Plugin Name
-    * Source Names (if applicable)
-    * Relative Path
+.. code-block:: bash
 
-For an example of how to run this command, see below, one of which applies this command
-to a specific package. Notice the use of aliases in case you want to use these commands
-in shorthand style. To see which packages are available, run ``geoips list packages``.
+    geoips list interfaces -i --package_name <package_name>
 
-::
+list interface
+^^^^^^^^^^^^^^
 
-    geoips ls alg
-    geoips ls algs
-    geoips list algorithm
-    geoips list algorithms
-    geoips list <interface_name> -p <package_name>
+``list <interface_name>`` is a sub-command of list.
 
-.. _geoips_list_interfaces:
+It returns a list of an interfaces plugins with the following plugin information:
 
-:ref:`geoips list interfaces <geoips_list_interfaces>`
+* GeoIPS Package
+* Interface Name
+* Interface Type
+* Family
+* Plugin Name
+* Source Names (if applicable)
+* Relative Path
 
-``list interfaces`` is a list sub-command which retrieves a listing of GeoIPS
-interfaces. This command has two modes; ``implemented`` and ``general``. Implemented
-mode searches for plugins of each interface which have been created throughout GeoIPS
-packages, or a certain package. General mode retrieves a listing of native GeoIPS
-Interfaces, which users can then create their own plugins using those interfaces.
-General Mode cannot be package specific.
-Information included when calling this command in implemented mode is:
+For example:
 
-    * GeoIPS Package
-    * Interface Type
-    * Interface Name
+.. code-block:: bash
 
-Information included when calling this command in general mode is:
+    geoips list algorithms
 
-    * GeoIPS Package
-    * Interface Type
-    * Interface Name
-    * Supported Families
-    * Docstring
-    * Absolute Path
+You can also filter by package name with ``--package_name`` or ``-p``. For example:
 
-For an example of how to run both modes of this command, see below.
+.. code-block:: bash
 
-Implemented Mode Options
-::
+    geoips list interfaces --package_name geoips
+    geoips list <interface_name> -p <package_name>
```

Comment:
Maybe this would be more clear if the example used a package other than ``geoips`` like ``geoips_clavrx``.

Reply:
Agreed.